js/leaflet/leaflet.js

```diff
@@ -1342,13 +1342,13 @@ var refreshLegend = function (_layer) {
         label6 = ['<strong> Weekly Change Rate of New Cases </strong>']
         label7 = ['<strong> Cases </strong>'];
 
-        // Changing the grades using unshift somehow also changes bins?
-        //grades.unshift(0);
         var legendContent = "";
 
         // loop through our density intervals and generate a label with a colored square for each interval
         if (_layer == il_county_case_layer_object || _layer == us_county_case_layer_object || _layer == us_state_case_layer_object || _layer == world_case_layer_object) {
             grades = bins;
+            // grades start from 0
+            grades.unshift(0);
```

Comment:
@lymine1996 
Zero value is still a separate level here https://github.com/cybergis/wherecovid19_webapp/blob/refactor/js/leaflet/leaflet.js#L976, so we should show it in the legend, something like 0, 1-XXXX, XXX-XXXX,,,,,,XXXXXX+